web/src/FacetsPane.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from "react"
+import { ReactComponent as LogoWordmarkSvg } from "./assets/svg/logo-wordmark-gray.svg"
+import AnsiLine from "./AnsiLine"
+import "./FacetsPane.scss"
+import { Resource } from "./types"
+
+type FacetsProps = {
+  resource: Resource
+}
+
+function logToLines(s: string) {
+  return s.split("\n").map((l, i) => <AnsiLine key={"logLine" + i} line={l} />)
+}
+
+class FacetsPane extends PureComponent<FacetsProps> {
+  render() {
+    let el = (
+      <section className="Pane-empty-message">
+        <LogoWordmarkSvg />
+        <h2>No Facets Found</h2>
+      </section>
+    )
+
+    let facets = this.renderFacets()
+    if (facets.length > 0) {
+      el = <ul>{facets}</ul>
+    }
+
+    return <section className="FacetsPane">{el}</section>
+  }
+
+  renderFacets(): Array<JSX.Element> {
+    if (!this.props.resource.facets) {
+      return new Array<JSX.Element>()
+    }
+    console.log("facets: " + JSON.stringify(this.props.resource.facets))
+    return this.props.resource.facets.map(facet => {
+      return (
+        <li className="FacetsPane-item">
+          <header>
+            <div className="FacetsPane-headerDiv">
+              <h3 className="FacetsPane-headerDiv-header">{facet.name}</h3>
```

Comment:
no big deal but i'd probably take out this class name and write a style rule for `div.FacetsPane-headerDiv h3` 🤷‍♀